base/interactiveutil.jl

```diff
@@ -439,7 +439,7 @@ function gen_call_with_extracted_types(__module__, fcn, ex0)
     ex = Meta.lower(__module__, ex0)
     exret = Expr(:none)
     if !isa(ex, Expr)
-        exret = Expr(:call, :error, "expression is not a function call or symbol")
+        return exret = Expr(:call, :error, "expression is not a function call")
```

Comment:
No need for the assignment with the immediate return, or am I missing something?

Reply:
fixed